### PR TITLE
Extend base templates for the homepage

### LINKED ISSUE #35: Extend base template in the ansible community theme for the homepage
For quick iteration we built the initial homepage "on top" of a Nikola site. Next step is to integrate the homepage that we built with the rest of our Nikola site. To do this, extend the base template with base_helper to include the compiled css and index templates.

This issue follows up on https://github.com/ansible-community/community-website/pull/32

- Add custom base templates to `themes/ansible-community` that include the redhat footer and compiled css.
- Build index.html off the base template.
- Replace `ansible.com` navbar with the Nikola theme navbar generated from `conf.py`
- Ensure posts and pages build as expected with the parent theme.

### PR DESCRIPTION
**Pull request by oraNod**
Fixes #35 and relates to #36

### CODE REVIEW

**Review by maxamillion — DISMISSED** [review]
This looks amazing! :+1:

**Review by samccann — APPROVED** [review]
Can't comment on the coding but generated locally and LGTM. I can shrink my browser down to 'iphone-like' size and the top nav bar behaves nicely

File: themes/ansible-community/templates/base.tmpl
Comment (GregSutcliffe) [inline themes/ansible-community/templates/base.tmpl:47]: Might be nice to have icon support here for the nav tuples, but I think we can come back to that.

**Review by GregSutcliffe — CHANGES_REQUESTED** [review]
Couple of minor things, mostly LGTM

File: themes/ansible-community/templates/base.tmpl
Comment (oraNod) [reply themes/ansible-community/templates/base.tmpl:47]: Yarp but I think we need to add that to the macro in the base_helper.tmpl file. I'll create an issue.

**Comment by oraNod** [comment]
@GregSutcliffe Not sure I like resetting approvals for new commits tbh.

File: themes/ansible-community/templates/base.tmpl
Comment (GregSutcliffe) [reply themes/ansible-community/templates/base.tmpl:47]: I tested it with only changing the conf.py and it works fine:
image

**Review by GregSutcliffe — APPROVED** [review]

**Comment by oraNod** [comment]
@maxamillion and @samccann The last push removed your approvals. Could you please take a look and re-approve? Thanks.

**Review by maxamillion — APPROVED** [review]
This is fantastic! +1

**Review by GregSutcliffe — APPROVED** [review]

**Comment by GregSutcliffe** [comment]
Great work, thanks @oraNod!

**Comment by oraNod** [comment]
I don't know, man. The current resetting approvals feels much too strict to me and adds unnecessary cycles. I understand the principle but this is the first repo where I've encountered the stale review setting and it's a little frustrating.

OK, so the last commit I pushed I probably could've done in a separate PR and I only have myself to blame. But the one before that was a commit for changes that you requested and was on top of the approval that Adam and Sandra had already given. Why should they need to re-approve for changes you requested?

**Comment by oraNod** [comment]
I agree that we need some safeguards but maybe stale approvals aren't the way to achieve that?

**Comment by GregSutcliffe** [comment]
>But the one before that was a commit for changes that you requested and was on top of the approval that Adam and Sandra had already given. Why should they need to re-approve for changes you requested?

Indeed, review changes would have to be approved, that would be expected. Why should they re-approve? Because you *might* have changed more than what was requested, or changed it in a way they did not like. They should get a chance to say so, rather than giving the *illusion* of approval for things they haven't actually seen.

I can't think of a different/better mechanism for making sure people have actually reviewed what's actually going to be deployed (rather than an older version that might not be relevant now). So the question is not different, it's are we going to relax the safeguards right now. I'm probably OK with that (we can log an issue to turn it back on), but I'd like to hear from @cybette on this.

**Comment by cybette** [comment]
I understand it can be a hassle and seems unnecessary to have to re-approve every change, but we should set a good example even at this stage, especially when we have a responsibility of the governance and publishing process for not just Red Hat but also the Community. I think what we can do is to remind and encourage maintainers/reviewers to get in the habit of re-checking changes and re-approving. I can take the responsibility to nag at people in the matrix room to do so, or can we make use of the GH bot somehow?

**Comment by GregSutcliffe** [comment]
Thanks @cybette. I think use of the "draft" flag for PRs is useful here - if we don't nag for approval until it's 90% of the way there (but still have the PR so we can gather more general feedback) then it should help?

**Pull request merged**
